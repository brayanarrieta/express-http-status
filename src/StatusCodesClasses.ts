/* eslint-disable */
/** 1×× Informational */
export const INFORMATIONAL = '1xx';
/** 2×× Success */
export const SUCCESS = '2xx';
/** 3×× Redirection */
export const REDIRECTION = '3xx';
/** 4×× Client Error */
export const CLIENT_ERROR = '4xx';
/** 5×× Server Error */
export const SERVER_ERROR = '5xx';
