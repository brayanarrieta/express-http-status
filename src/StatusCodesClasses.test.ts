import * as StatusCodesClasses from './StatusCodesClasses';

describe('StatusCodesClasses', () => {
  it('should export the correct structure', () => {
    expect(StatusCodesClasses).toMatchSnapshot();
  });
});
