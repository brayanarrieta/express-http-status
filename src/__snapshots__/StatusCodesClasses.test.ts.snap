// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatusCodesClasses should export the correct structure 1`] = `
Object {
  "CLIENT_ERROR": "4xx",
  "INFORMATIONAL": "1xx",
  "REDIRECTION": "3xx",
  "SERVER_ERROR": "5xx",
  "SUCCESS": "2xx",
}
`;
